.github/workflows/reusable-audit.yml

```
@@ -0,0 +1,46 @@
+name: Reusable Audit Workflow
+
+on:
+  workflow_call:
+    inputs:
+      useForceResolutions:
+        required: false
+        default: false
+        type: boolean
+    secrets:
+      WEBHOOK_E2E_FAILURE:
+        required: true
+
+jobs:
+  audit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.4.0
+      - uses: actions/setup-node@v2.5.1
+        with:
+          node-version: "16.x"
+          registry-url: "https://registry.npmjs.org"
+      - run: npm ci
+      # We check the dependency licenses as an audit task:
```

```suggestion
      # We check the dependency licenses as part of the audit job, as a
      # misconfigured or inappropriate dependency license is in the same
      # category of issues as an npm audit security failure: we consider 
      # bad licenses a vulnerability.```

How's this change @ajacksified ?

---

@NSeydoux maybe take this change?